Author: Daniel Baumann <daniel.baumann@progress-technologies.net>
Description: Update gnuefi file locations.

diff -Naurp efilinux.orig/Makefile efilinux/Makefile
--- efilinux.orig/Makefile	2012-05-18 14:16:41.965605197 +0200
+++ efilinux/Makefile	2012-05-18 14:21:30.431531248 +0200
@@ -51,8 +51,8 @@ endif
 INCDIR := /usr/include
 
 # gnuefi sometimes installs these under a gnuefi/ directory, and sometimes not
-CRT0 := $(shell find $(LIBDIR) -name crt0-efi-$(ARCH).o 2>/dev/null | tail -n1)
-LDSCRIPT := $(shell find $(LIBDIR) -name elf_$(ARCH)_efi.lds 2>/dev/null | tail -n1)
+CRT0 := $(LIBDIR)/crt0-efi-$(ARCH).o
+LDSCRIPT := $(LIBDIR)/elf_$(ARCH)_efi.lds
 
 CFLAGS=-I. -I$(INCDIR)/efi -I$(INCDIR)/efi/$(ARCH) \
 		-DEFI_FUNCTION_WRAPPER -fPIC -fshort-wchar -ffreestanding \
